src/libraries/System.Runtime/tests/System.Globalization.Tests/DateTimeFormatInfo/DateTimeFormatInfoShortDatePattern.cs

```diff
@@ -33,7 +33,7 @@ public static IEnumerable<object[]> ShortDatePattern_Get_TestData_HybridGlobaliz
             yield return new object[] { "en-AG", "dd/MM/yyyy" };
             yield return new object[] { "en-AI", "dd/MM/yyyy" };
             yield return new object[] { "en-AS", "M/d/yyyy" };
-            yield return new object[] { "en-AT", "dd/MM/yyyy" };
+            yield return new object[] { "en-AT", PlatformDetection.IsHybridGlobalizationOnBrowser ? "dd/MM/yyyy" : "dd.MM.yyyy" }; // "dd/MM/yyyy"
```

Comment:
These differences were there even before we started normalizing? I don't see a place where Normalize/Fix methods would change `/` to `.`

Reply:
Yes, this comes with "." from the Apple ICU.